packages/backend/src/accounting/service.test.ts

```diff
@@ -36,9 +33,9 @@ describe('Accounting Service', (): void => {
   }
 
   beforeAll(async (): Promise<void> => {
-    tigerbeetleContainer = await startTigerbeetleContainer()
+    tigerbeetleContainer = await startTigerbeetleContainer({})
     Config.tigerbeetleReplicaAddresses = [
-      tigerbeetleContainer.getMappedPort(TIGERBEETLE_PORT)
+      tigerbeetleContainer.getMappedPort(Config.tigerbeetlePort)
```

Comment:
Could `startTigerbeetleContainer` also return this, so that callers don't need to use `Config.tigerbeetlePort`?
Alternatively `tests/tigerbeetle.ts` could also export a
```
getAddress(container: StartedTestContainer): string
```